tests/spec/fs.copyFile.spec.js

```diff
@@ -0,0 +1,52 @@
+var util = require('../lib/test-utils.js');
+var expect = require('chai').expect;
+
+describe('fs.copyFile', function(){
+  beforeEach(function(done){
+    util.setup(function() {
+      var fs = util.fs();
+      fs.writeFile('/srcfile', 'This is a src file.', function(error){
+        if(error) throw error;
+        fs.writeFile('/destfile', 'This is a dest file.', function(error){
+          if(error) throw error;
+          done();
+        });
+      });
+    });
+  });
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fs = util.fs();
+    expect(fs.copyFile).to.be.a('function');
+  });
+
+  it('should return an error if the src path does not exist', function(done){
+    var fs = util.fs();
+    var src = null;
+    var dest = 'dest.txt';
+
+    fs.copyFile(src, dest, function(error){
+      expect(error).to.exist;
+      expect(error.code).to.equal('ENOENT');
+      done();
+    });
+   
+  });
+
+  it('should copy file successfully', function(done) {
```

Comment:
This tests if `/destfile` has been overwritten with the contents of `/srcfile`, which is great! What about the case where `/destfile` does not already exist?

Reply:
Add some extra info to line 37 to indicate `...when destination does not exist`